tests/unit/admin/test-class-wc-payments-admin.php

```diff
@@ -280,7 +280,7 @@ public function test_maybe_redirect_overview_to_connect( $expected_times_redirec
 				'position'   => '55.7', // After WooCommerce & Product menu items.
 				'icon'       => '',
 				'nav_args'   => [
-					'title'        => __( 'WooCommerce Payments', 'woocommerce-payments' ),
+					'title'        => __( 'WooPayments', 'woocommerce-payments' ),
```

Comment:
No need to translate here.

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/6581/commits/b1d36c896aa92104e381f3d5d35fad7804264515